apps/studio/components/interfaces/Settings/Database/DatabaseSettings/DatabaseConnectionString.tsx

```diff
@@ -1,28 +1,57 @@
 import { useParams, useTelemetryProps } from 'common'
+import dayjs from 'dayjs'
 import { useRouter } from 'next/router'
-import { useEffect, useRef } from 'react'
-import { Input, Tabs } from 'ui'
-
+import { useEffect, useRef, useState } from 'react'
+import {
+  AlertDescription_Shadcn_,
+  AlertTitle_Shadcn_,
+  Alert_Shadcn_,
+  Badge,
+  Button,
+  IconAlertTriangle,
+  IconExternalLink,
+  Input,
+  Tabs,
+} from 'ui'
+
+import { useProjectContext } from 'components/layouts/ProjectLayout/ProjectContext'
 import AlertError from 'components/ui/AlertError'
 import DatabaseSelector from 'components/ui/DatabaseSelector'
 import Panel from 'components/ui/Panel'
 import ShimmeringLoader from 'components/ui/ShimmeringLoader'
 import { useProjectSettingsQuery } from 'data/config/project-settings-query'
 import { useReadReplicasQuery } from 'data/read-replicas/replicas-query'
-
 import { useFlag } from 'hooks'
 import { pluckObjectFields } from 'lib/helpers'
 import Telemetry from 'lib/telemetry'
 import { useDatabaseSelectorStateSnapshot } from 'state/database-selector'
-
-const DatabaseConnectionString = () => {
+import { getConnectionStrings } from './DatabaseSettings.utils'
+
+const CONNECTION_TYPES = [
+  { id: 'psql', label: 'PSQL' },
+  { id: 'uri', label: 'URI' },
+  { id: 'golang', label: 'Golang' },
+  { id: 'jdbc', label: 'JDBC' },
+  { id: 'dotnet', label: '.NET' },
+  { id: 'nodejs', label: 'Nodejs' },
+  { id: 'php', label: 'PHP' },
+  { id: 'python', label: 'Python' },
+]
+
+export const DatabaseConnectionString = () => {
   const router = useRouter()
+  const { project: projectDetails } = useProjectContext()
   const { ref: projectRef, connectionString } = useParams()
   const telemetryProps = useTelemetryProps()
   const state = useDatabaseSelectorStateSnapshot()
 
-  const readReplicasEnabled = useFlag('readReplicas')
+  // [Joshen] TODO this needs to be obtained from BE as 26th Jan is when we'll start - projects will be affected at different rates
+  const resolvesToIpV6 = false // Number(new Date()) > Number(dayjs.utc('01-26-2024', 'MM-DD-YYYY').toDate())
```

Comment:
Just confirming this should be hardcoded to `false` for now?

Reply:
yup - i'm checking with BE whether there'll be some sort of project level flag, but we can update this logic in a separate PR so the warnings can go through first